src/CMakeLists.txt

```diff
@@ -50,6 +49,7 @@ if (USE_CUDA)
     CUDA_STANDARD 14
     CUDA_STANDARD_REQUIRED ON
     CUDA_SEPARABLE_COMPILATION OFF)
+  target_link_libraries(objxgboost PRIVATE GPUTreeShap::GPUTreeShap)
```

Comment:
```suggestion
  target_link_libraries(objxgboost PUBLIC GPUTreeShap::GPUTreeShap)
```

Reply:
This ensures that the GPU shap header is accessible to the gtest binary.

Reply:
I thought it should be private based on this: https://cmake.org/pipermail/cmake/2016-May/063400.html

Reply:
Never mind. tests/cpp/predictor/test_gpu_predictor.cu doesn't directly access `GPUTreeShap/gpu_treeshap.h`